[2023-02-07T18:32:03.684+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.download_local_and_upload_GCS.download_covid_cases_dataset_task scheduled__2023-02-06T00:00:00+00:00 [queued]>
[2023-02-07T18:32:03.701+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.download_local_and_upload_GCS.download_covid_cases_dataset_task scheduled__2023-02-06T00:00:00+00:00 [queued]>
[2023-02-07T18:32:03.702+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-02-07T18:32:03.702+0000] {taskinstance.py:1280} INFO - Starting attempt 3 of 4
[2023-02-07T18:32:03.702+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-02-07T18:32:03.724+0000] {taskinstance.py:1300} INFO - Executing <Task(BashOperator): download_local_and_upload_GCS.download_covid_cases_dataset_task> on 2023-02-06 00:00:00+00:00
[2023-02-07T18:32:03.740+0000] {standard_task_runner.py:55} INFO - Started process 2429 to run task
[2023-02-07T18:32:03.753+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag', 'download_local_and_upload_GCS.download_covid_cases_dataset_task', 'scheduled__2023-02-06T00:00:00+00:00', '--job-id', '64', '--raw', '--subdir', 'DAGS_FOLDER/covid19_pipeline.py', '--cfg-path', '/tmp/tmpqxcpyuvo']
[2023-02-07T18:32:03.754+0000] {standard_task_runner.py:83} INFO - Job 64: Subtask download_local_and_upload_GCS.download_covid_cases_dataset_task
[2023-02-07T18:32:03.919+0000] {task_command.py:388} INFO - Running <TaskInstance: data_ingestion_gcs_dag.download_local_and_upload_GCS.download_covid_cases_dataset_task scheduled__2023-02-06T00:00:00+00:00 [running]> on host 2231c3c92ba2
[2023-02-07T18:32:04.035+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag
AIRFLOW_CTX_TASK_ID=download_local_and_upload_GCS.download_covid_cases_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2023-02-06T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=3
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-02-06T00:00:00+00:00
[2023-02-07T18:32:04.043+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-02-07T18:32:04.044+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'curl -sS https://covid19.who.int/WHO-COVID-19-global-data.csv > /opt/***/covid_cases.csv']
[2023-02-07T18:32:04.069+0000] {subprocess.py:86} INFO - Output:
[2023-02-07T18:32:05.026+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-02-07T18:32:05.087+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag, task_id=download_local_and_upload_GCS.download_covid_cases_dataset_task, execution_date=20230206T000000, start_date=20230207T183203, end_date=20230207T183205
[2023-02-07T18:32:05.133+0000] {local_task_job.py:208} INFO - Task exited with return code 0
